satellite/metainfo/metainfo.go

```diff
@@ -147,6 +143,10 @@ func (endpoint *Endpoint) CreateSegmentOld(ctx context.Context, req *pb.SegmentW
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
 
+	if !req.Expiration.IsZero() && !req.Expiration.After(time.Now()) {
```

Comment:
should we do UTC ?

Reply:
internally time.Time will handle timezones